app/config/security.yml

```diff
@@ -93,15 +103,15 @@ security:
         # /admin accessible only from ADMIN, super users and AD users
         - { path: ^/admin, roles: [ROLE_ADMIN, ROLE_AD] }
         # client settings not available to PA users
-        - { path: ^/deputyship-details, roles: [ROLE_DEPUTY, ROLE_ADMIN, ROLE_AD] }
+        - { path: ^/deputyship-details, roles: [ROLE_LAY_DEPUTY, ROLE_ADMIN, ROLE_AD] }
         - { path: ^/user-account/, roles: [ROLE_DEPUTY, ROLE_ADMIN, ROLE_AD] }
         - { path: ^/ad, roles: [ROLE_AD] }
-        - { path: ^/pa/team/add, roles: [ROLE_PA, ROLE_PA_ADMIN] }
-        - { path: ^/pa/team/edit, roles: [ROLE_PA, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER] }
-        - { path: ^/pa/team, roles: [ROLE_PA, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER] }
-        - { path: ^/pa/, roles: [ROLE_PA, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER] }
+        - { path: ^/pa/team/add, roles: [ROLE_PA_NAMED, ROLE_PA_ADMIN] }
+        - { path: ^/pa/team/edit, roles: [ROLE_PA] }
+        - { path: ^/pa/team, roles: [ROLE_PA] }
```

Comment:
ROLE_PA_NAMED here too ?

Reply:
need to fix team things. initially under ROLE_ORG

Reply:
ROLE_PA now means ROLE_PA_*